thinc/backends/numpy_ops.pyx

```diff
@@ -523,9 +523,10 @@ def check_seq2col_lengths(ops, lengths, B):
 
 
 def cpu_clip_gradient(weight_t[::1] gradient, weight_t threshold):
-    grad_norm = Vec.norm(&gradient[0], gradient.shape[0])
+    arr_view = numpy.asarray(<np.float32_t[:gradient.shape[0]]> &gradient[0])
+    grad_norm = numpy.linalg.norm(arr_view)
```

Comment:
Can we use BLAS `snrm2` here to avoid rewrapping the array using numpy? Maybe that'd also allow making it `cdef nogil`?

Reply:
Actually, that entire function is unused/vestigial and can/should be removed in `v9.0`. The default `Ops` class handles the gradient clipping in its [`clip_gradient` method](https://github.com/explosion/thinc/blob/eda4c75771667aee114e640b9c1e4021a514a275/thinc/backends/ops.py#L1158). `CupyOps` overrides this, but `NumpyOps` doesn't.